test/alternator/test_returnvalues.py

```diff
@@ -124,6 +165,41 @@ def test_delete_item_returnvalues(test_table_s):
     with pytest.raises(ClientError, match='ValidationException'):
         test_table_s.delete_item(Key={'p': p}, ReturnValues='none')
 
+# Testing ReturnValuesOnConditionCheckFailure feature which returns values only
+# on failed condition expression.
```

Comment:
Nitpick - maybe mention the operation type in each of these identical comments - e.g., here say " ... on failed condition expression in DeleteItem operation", and in the previous test, it's the PutItem operation, and so on.

Reply:
what would be the benefit of such addition?

Reply:
Just a better explanation of what this specific test does. How can several tests have exactly the same comment explaining what they do? :-)

Reply:
I doubt anyone would find this out :) But my understanding is that primary description is entity's name (i.e. test name, function name, variable name) and comment provides additional info, it may even formulate the same thing using different wording - in case someone didn't understand. I see little value in having something like:

```
# Tests put item on error.
def test_put_item_on_error(...)
```

Reply:
The example you gave is irrelevant - both because in that example you really copied the function's name into a comment, which I'm not suggesting to do, and because you have a 5-word comment, which I also don't want you do - I want descriptions of tests to be multiple lines. You don't know how difficult it is to debug test failures when the test author didn't bother to explain why he thought this test is correct or relevant or what it's even supposed to test.

Anyway, your actual code is:
```
# Testing ReturnValuesOnConditionCheckFailure feature which returns values only
# on failed condition expression.
def test_delete_item_returnvalues_on_condition_check_failure(test_table_s):
```
Why do you want to "hide" the fact that this test is specific to DeleteItem? Yes, the name of the test has the words "delete_item". But it also has 6 other words. It's not impossible to figure out, but isn't it much easier just to say what this test actually does (and not have 3 different tests with exactly the same comment) instead of making it into a riddle? :-)

Anyway, I don't insist, I'll merge as-is.